bucketeer/src/main/kotlin/io/bucketeer/sdk/android/internal/remote/ApiClientImpl.kt

```diff
@@ -119,7 +120,7 @@ internal class ApiClientImpl(
   }
 
   override fun registerEvents(events: List<Event>): RegisterEventsResult {
-    val body = RegisterEventsRequest(events = events)
+    val body = RegisterEventsRequest(events = events, sourceId = SourceID.ANDROID)
```

Comment:
Just confirming.
Is it necessary to pass the sourceID here?

Like the `sdkVersion`, the `sourceID` is set by default in the data class.
https://github.com/bucketeer-io/android-client-sdk/pull/165/files#diff-4d7bb55f81bf07536550c06fc990b013fd606b6054932e3c88bb9c60ee44cbd4R12

Reply:
no, It's not necessary. I removed it.